linters/djlint/README.md

```diff
@@ -0,0 +1,10 @@
+# DJLint
+
+- [README](https://github.com/Riverside-Healthcare/djlint#readme)
+- [Rules](https://www.djlint.com/docs/linter/)
+
+DJLint supports Django • Jinja • Nunjucks • Twig • Handlebars • Mustache • GoLang • Angular
+
+For use of anything other than Django, please visit the readme and set the appropriate profile.
```

Comment:
Good callout. Can you add "set the appropriate profile in your `run` command or in the linter config"?

As another note, in the future, we will likely add command-specific filetypes and hopefully we can do this sort of profile management automatically within the same linter.